app/src/main/java/org/thoughtcrime/securesms/ui/components/ButtonSize.kt

```diff
@@ -0,0 +1,40 @@
+package org.thoughtcrime.securesms.ui.components
+
+import android.annotation.SuppressLint
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.LocalMinimumInteractiveComponentEnforcement
+import androidx.compose.material.LocalTextStyle
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import org.thoughtcrime.securesms.ui.baseBold
+import org.thoughtcrime.securesms.ui.extraSmallBold
+
+interface ButtonSize {
```

Comment:
Should this be ButtonStyle? It contains for size and text style.

Reply:
ButtonSize is confusing as the class affects the TextStyle.
ButtonStyle feels more global.

Reply:
@bemusementpark

Reply:
done